poetro_app/lib/features/poetry/presentation/widgets/poetry_previw_item.dart

```diff
@@ -1,25 +1,25 @@
 import 'package:flutter/material.dart';
 import 'package:go_router/go_router.dart';
-import 'package:poetro_app/config/routes/routes.dart';
+import 'package:poetro_app/config/routes/app_routes.dart';
 import 'package:poetro_app/core/const/app_colors.dart';
 import 'package:poetro_app/features/poetry/presentation/models/poetry_model.dart';
 
 class PoetryPreviwItem extends StatelessWidget {
-  final PoetryModel poetryDTO;
+  final PoetryModel poetryModel;
 
-  const PoetryPreviwItem({super.key, required this.poetryDTO});
+  const PoetryPreviwItem({super.key, required this.poetryModel});
 
   @override
   Widget build(BuildContext context) {
     return Card(
       color: AppColors.primaryColor.withOpacity(0.2),
```

Comment:
use only inherited widget for theming or other provider in real project, to simplift in future adding new themes or change previous one